src/content/advanced/ingress/multi-nginx-ic/index.md

```diff
@@ -113,6 +113,20 @@ For NGINX IC running on on-prem (KVM) workload clusters there's no out-of-the-bo
 
 More information on this topic can be found in document [Services of type LoadBalancer]({{< relref "/content/advanced/ingress/service-type-loadbalancer/index.md" >}}).
 
+On AWS and Azure, it is also possible to only install a single Nginx Ingress Controller App and to delegate both internal and external traffic to it. Here is a minimal working example on how to achieve this goal.
```

Comment:
Theoretically this is also possible on-prem but just results in two different `NodePort` services. This way customers can configure a public and an internal load balancer in front of the workload cluster on their own.

Reply:
Done